docker/README.md

```diff
@@ -0,0 +1,54 @@
+--------
+# Docker for BigDL
```

Comment:
Docker images and builders for BigDL

Reply:
Add descriptions for the folders and scripts contained in the image.

Reply:
ok.